docs/topic/credits.md

```diff
@@ -0,0 +1,48 @@
+# Cloud Credits
+
+On each cloud provider, it is common to provide cloud credits that cover the infrastructure costs instead of paying with cash.
+This page has information about how this works for each major cloud provider we use.
+
+## Amazon Web Services
+
+On Amazon Web Services, credits come via **Promotion Codes**.
+These are unique strings that are attached to a given credit offer.
+We can apply these codes to any Account in AWS.
+To do so, follow these steps:
+
+1. Log in to the relevant account via [our AWS SSO](cloud-access:aws-sso).
```

Comment:
Add a note that credits only apply from the *next billing period*, so next calendar month?